updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0
+
 :: Update hosts file
-python updateHostsFile.py -a
+if exist updateHostsFile.exe (
+    echo run exe
+    updateHostsFile --auto --extensions fakenews social gambling
+) else (
+    echo run script
+    python updateHostsFile.py --auto --extensions fakenews social gambling
+)
 
 :: Move new hosts file in-place
 COPY hosts %WINDIR%\System32\drivers\etc\
 
 :: Flush the DNS cache
 ipconfig /flushdns
+
+:endofscript

```

Comment:
Can you explain this to me? I'm not a windows user but it looks strange that this "appears" from nowhere

Reply:
`:endofscript` is a batch file label and I added it as part of my testing/development. It allowed me to put `goto endofscript` after the part I was testing without having to comment out or mess with the rest of the lines.

Reply:
So not needed in our scale.